core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -1052,7 +1053,9 @@ class KafkaController(val config: KafkaConfig, zkClient: KafkaZkClient, time: Ti
       debug(s"Live brokers: ${controllerContext.liveBrokerIds.mkString(",")}")
 
       val partitionsToActOn = controllerContext.partitionsOnBroker(id).filter { partition =>
-        controllerContext.partitionReplicaAssignment(partition).size > 1 && controllerContext.partitionLeadershipInfo.contains(partition)
+        controllerContext.partitionReplicaAssignment(partition).size > 1 &&
+          controllerContext.partitionLeadershipInfo.contains(partition) &&
```

Comment:
Could we fix the alignment?

Reply:
Can you be more specific? If you mean the indentation by 2 spaces in the trailing conditions, it seems that's the correct alignment, as also being used in line 436 and line 1172 of this file.

Reply:
Sorry. If there is the standard, we can just leave it as it is.